pages/api/helpjuice/[...path].page.ts

```diff
@@ -0,0 +1,48 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import fetch from 'node-fetch';
+
+export const buildUrlPath = (originalPath: string | string[] | undefined) => {
+  if (!originalPath) {
+    return originalPath;
+  }
+  let pathArray;
+  if (typeof originalPath === 'string') {
+    pathArray = originalPath.split(',');
+  } else {
+    pathArray = originalPath;
+  }
+  return pathArray.join('/');
+};
+
+const proxyCatchAll = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+): Promise<void> => {
+  try {
+    const allQueryParams = req.query;
+    const path = req.query.path;
+    delete allQueryParams.path;
```

Comment:
Instead of modifying data you can use this technic to get the path and get all the other properties but the path.

```suggestion
    const { path, ...allQueryParams } = req.query;
```